eth-node/bridge/geth/waku.go

```diff
@@ -231,7 +241,7 @@ func (w *gethWakuWrapper) SendMessagesRequest(peerID []byte, r types.MessagesReq
 		Limit:  r.Limit,
 		Cursor: r.Cursor,
 		Bloom:  r.Bloom,
-		Topics: r.Topics,
+		Topics: r.ContentTopics,
```

Comment:
Does Waku V1 support `ContentTopics`?

Reply:
WakuV2 content topics and WakuV1 topics represent the same concept (a 4 bytes value)